archey/entry.py

```diff
@@ -26,7 +27,10 @@ def __init__(self, name: Optional[str] = None, value=None, options: Optional[dic
         # `name`: key (defaults to the instantiated entry class name);
         # `value`: value of entry as an appropriate object;
         # `options`: configuration options *specific* to an entry instance;
+
         self.name = name or self._PRETTY_NAME or self.__class__.__name__
+        if self._ICON is not None and Configuration().get("entries_icon"):
```

Comment:
```suggestion
        if self._ICON is not None and configuration.get("entries_icon"):
```

(as well as `self._default_strings = configuration.get("default_strings")` below, but GitHub reviewing UI won't let me suggest it)

Reply:
Makes more sense. I missed the one underneath, my bad!